### DISCUSSION TITLE
Is it possible to create a bot and run it simultaneously on 3 virtual machines?

### DISCUSSION THREAD

**Discussion by a1sbes** [aiogram/aiogram#1216] [General]
If it is possible to create a bot and run it simultaneously on 3 virtual machines to distribute the load, can you write an example?

**Comment by Olegt0rr** [comment]
1. Your app should use webhooks
2. Your app should be stateless
3. Use any proxy balancer to distribute Telegram Updates between your instances (nginx, traefik, e.t.c.) or use DNS roundrobin.

Thats all :)
